modules/tile-converter/docs/api-reference/tile-converter-bundle.md

```diff
@@ -0,0 +1,39 @@
+## Tile converter bundle script
```

Comment:
I think one page for building the bundle is enough, you can join these two pages.

Reply:
The first page - `npx tile-converter` + running I3S local server. It can be used for running converter on latest master branch.  The second page - `node converter.min.js`. It can be used for fast distribution to not well experienced users. In my opinion, separate instruction for this case might be helpful.

Reply:
OK. Please explain these things at the top of respective page - it is hard for a user to understand what they need to read and what they can skip